### PR TITLE
[10]Create and Style Customer Sign Up page

### PR DESCRIPTION
**Pull request by Fiyin985**
Screenshot (27)

### CODE REVIEW

**Comment by Fiyin985** [comment]
ive done it

**Comment by adaugochi** [comment]
I don't think you did, I am still seeing this 
image

**Comment by Fiyin985** [comment]
I removed that picture

File: submissions/fiyingabriel/css/style.css [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+}
+main .heading h1{
+ color: #004b18;
+}
+main .existed{
+ margin-bottom: 1em;
+ text-align: center;
+ font-size: 1rem;
+}
+main form{
```
Comment (adaugochi) [inline submissions/fiyingabriel/css/style.css:58]: It is not advisable to style HTML element themself. give it a class name or id. What if you have form tag elsewhere on that same page?

same goes to other places where you are style HTML element. expect if you want that element to have that style generally. E.g removing the underline on an anchor tag, `<a>` tag.  or removing the default margin and padding a body tag has

**Pull request merged**